server/sonar-server/src/main/java/org/sonar/server/webhook/WebhookPayloadFactory.java

```diff
@@ -17,13 +17,11 @@
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
-package org.sonar.server.computation.task.projectanalysis.webhook;
-
-import org.sonar.api.ce.posttask.PostProjectAnalysisTask;
+package org.sonar.server.webhook;
 
 @FunctionalInterface
 public interface WebhookPayloadFactory {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Drop this interface in favor of "java.util.function.Function<ProjectAnalysis,WebhookPayload>". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1711)